js/skip-nav.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/* global require, module, document */
+
+var $ = require('jquery');
+
+function Skipnav(selector) {
```

Comment:
Could you add a brief description of what this is here? The name "Skipnav" is a little vague for me. A brief jsdoc-style comment would be great, a la https://github.com/18F/fec-style/blob/master/js/dropdowns.js#L11-L15.